www/codemirror/codemirror-editTextSelection.js

```diff
@@ -110,7 +110,7 @@ const getWikitextRangeToSelect = (wikitextBeforeSelection, wikitextSelection) =>
 }
 
 const scrollToAndHighlightRange = (range, codemirror) => {
-  codemirror.setCursor(range.to)
+  codemirror.setSelection(range.from, range.to, {scroll: false})
 
   let marker = null
   marker = codemirror.markText(range.from, range.to, {
```

Comment:
`markText` can be removed since the text is highlighted by the selection now

Reply:
makes sense to me but given https://phabricator.wikimedia.org/T215448#5012000 - maybe we can remove it after confirming with Carolyn? I added a comment to the ticket https://phabricator.wikimedia.org/T219521#5075653